apis/python/src/tiledbsoma/_dataframe.py

```diff
@@ -270,18 +270,16 @@ def count(self) -> int:
             Experimental.
         """
         self._check_open_read()
-        return cast(int, self._soma_reader().nnz())
+        handle: clib.DataFrameWrapper = self._handle
+        return cast(int, handle.count)
```

Comment:
Could some of these be avoided by making a `libtiledbsoma.pyi` file? That would also give us some tooling help in editors. Up to you, and even if you do want to, it’s not something that needs to happen in this change.